packages/eslint-plugin/src/rules/use-component-selector.ts

```diff
@@ -33,6 +33,10 @@ export default createESLintRule<Options, MessageIds>({
           return;
         }
 
+        if (selector && ASTUtils.isTemplateLiteral(selector)) {
```

Comment:
Could this check just be added to the original if statement? Also, would you be able to add a test case or two to ensure this works?